insights/combiners/cryptsetup.py

```diff
@@ -0,0 +1,80 @@
+"""
+Cryptsetup - combine metadata about LUKS devices
+================================================
+
+Use LuksDump parser to filter just LUKS version 1 devices and return their
+paths (by UUID). Also combine outputs of LuksDump and LuksMeta parsers (with
+the same UUID) into a single LuksDevice.
+"""
+
+import copy
+
+from insights import SkipComponent
+from insights.core.plugins import combiner
+from insights.parsers.cryptsetup_luksDump import LuksDump
+from insights.parsers.luksmeta import LuksMeta
+
+
+@combiner(LuksDump)
```

Comment:
@danzatt - from my perspective, It seems this combiner function is not so required. Firstly, it's based on the single parser, if really want such an interface to return the "luks1 block devices", you can totally implement it in the `LuksDump` parser itself. Or, you can move this check into the `LuksDevices` combiner as a double safety-check here L77: `if luks_metas and uuid in luksmeta_by_uuid:`